admin_guide/manage_scc.adoc

```diff
@@ -429,56 +429,59 @@ To provide additional capabilities:
 === Modify Cluster Default Behavior
 
 To modify your cluster so that it does not pre-allocate UIDs, allows containers
-to run as any user, and prevents privileged containers:
+to run as any user, but prevents privileged containers, grant access to the
+*anyuid* SCC for everyone:
```

Comment:
@kalexand-rh This feels like a comma splice (before your changes, too!). Not sure how to fix.
"You can configure your cluster to not pre-allocate UIDs, which (?) allows containers to run as any user, but prevents privileged containers.  To configure this, grant access to the *anyuid* SCC for all users."
Something like that to offset  _grant access_ (above it almost sounds like granting access to everyone is another result of preventing pre-allocation.

Reply:
I don't think it's a comma splice, but the initial clause is way overloaded. I think it's trying to say, "To modify your cluster so that *all the things*, grant access to the *anyuid* SCC for everyone."

Maybe it should say:
```
When you grant access to the *anyuid* SCC for everyone, your cluster:
* Does not pre-allocate UIDs
* Allows containers to run as any user
* Prevents privileged containers
```

Reply:
@kalexand-rh That looks better!